### PR TITLE
Added the GitHub Workflows badges

### PR DESCRIPTION
**Pull request by jhsu802701**

### CODE REVIEW

**Comment by jhsu802701** [comment]
This addresses Issue #2670 .

File: README.md [hunk 1]
```diff
@@ -25,13 +25,17 @@ Additionally, [RubyTogether](https://rubytogether.org) sponsors individuals to w
 * [Mailing List][]
 * [FAQ][]
 * [IRC][]: #rubygems on Freenode
+* [GitHub Workflow][]: [![test workflow](https://github.com/rubygems/rubygems.org/actions/workflows/test.yml/badge.svg)](https://github.com/rubygems/rubygems.org/actions/workflows/test.yml)
+* [GitHub Workflow][]: [![lint workflow](https://github.com/rubygems/rubygems.org/actions/workflows/lint.yml/badge.svg)](https://github.com/rubygems/rubygems.org/actions/workflows/lint.yml)
+* [GitHub Workflow][]: [![docker workflow](https://github.com/rubygems/rubygems.org/actions/workflows/docker.yml/badge.svg)](https://github.com/rubygems/rubygems.org/actions/workflows/docker.yml)
```
Comment (sonalkr132) [inline README.md:30]: perhaps a single item with three badges would be more suitable here?

Comment (jhsu802701) [reply README.md]: OK, I consolidated the three badges into one line.

**Comment by sonalkr132** [comment]
Thank you @jhsu802701 :star:

**Pull request merged**